sdk/core/System.ClientModel/tests/Pipeline/HttpClientPipelineTransportTests.cs

```diff
@@ -353,7 +353,7 @@ public async Task UnbufferedResponseContentStreamContainsResponseContent()
         Assert.AreEqual("Mock content", BinaryData.FromStream(message.Response!.ContentStream!).ToString());
 
         // The second time it's read the stream's Position is at the end
-        Assert.AreEqual(string.Empty, BinaryData.FromStream(message.Response!.ContentStream!).ToString());
+        Assert.AreEqual(0, BinaryData.FromStream(message.Response!.ContentStream!).ToArray().Length);
```

Comment:
Curious what makes this needed?  (No pushback at all, truly just interested to understand better)

Reply:
This is also because of the bug in System.Memory.Data - https://github.com/dotnet/runtime/issues/68262 where calling ToString on an empty binary data throws an exception instead of returning string.Empty. As far as I know this bug only exists in 6.0.0 so it's more of a temp workaround. I can add this to the list of things to switch back once the package is upgraded again if it seems like a better test to check ToString.